syntax = "proto3";

package Persistance;

import "Vector.proto";

message SceneSetup {
	string cloud_path = 1;
	float cloud_size_m = 2;
	Persistance.Vector3 light_direction = 3;
}